net-libs/libwebsockets/metadata.xml

```diff
@@ -19,18 +19,26 @@
 	</maintainer>
 	<use>
 		<flag name="access-log">Support generating Apache-compatible access logs</flag>
+		<flag name="caps">Use Linux capabilities library to control privilege</flag>
 		<flag name="cgi">Include CGI (spawn process with network-connected stdin/out/err) APIs</flag>
+		<flag name="dir">Enable directory scanning api support</flag>
+		<flag name="file-ops">Support file operations vfs</flag>
 		<flag name="client">The client part of the library and libwebsockets-test-client</flag>
 		<flag name="generic-sessions">With the Generic Sessions plugin</flag>
 		<flag name="http2">Support the HTTP/2 protocol</flag>
 		<flag name="http-proxy">Support for rewriting HTTP proxying</flag>
+		<flag name="http-uncommon-headers">Inlude less common http header support</flag>
 		<flag name="lejp">With the Lightweight JSON Parser</flag>
 		<flag name="libev">Support event loops via dev-libs/libev</flag>
 		<flag name="libevent">Support event loops via dev-libs/libevent</flag>
 		<flag name="libuv">Support asynchronous I/O via dev-libs/libuv</flag>
 		<flag name="peer-limits">Track peers and restrict resources a single peer can allocate</flag>
 		<flag name="server-status">Support json + jscript server monitoring</flag>
 		<flag name="smtp">Provide SMTP support</flag>
+		<flag name="ssl">Include SSL support (defaults to OpenSSL)</flag>
+		<flag name="libressl">Use LibreSSL replacement for OpenSSL</flag>
+		<flag name="mbedtls">Use mbedTLS replacement for OpenSSL</flag>
+		<flag name="testapps">Don't build the libwebsocket-test-apps</flag>
```

Comment:
Fixed it, I also removed + from `testapps` USE flags since test-apps are disabled by default (since I added `-DLWS_WITHOUT_TESTAPPS=$(usex !testapps)`)